File: PrimeNumber/PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber{

    public static boolean isPrime(int num){
        for(int i = 2; i< num/2; i++){
            if(num%i == 0){
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number: ");
        int number = scanner.nextInt();

        if(isPrime(number)){
            System.out.println(number + " is a prime number.");
        }else{
            System.out.println(number + " is not a prime number.");
        }
    }

    
}

File: PrimeNumber/PrimeNumber.js
const isPrime = (num) => {
  for (let i = 2; i < num / 2; i++) {
    if (num % i === 0) {
      return false;
    }
  }
  return true;
};

function PrimeNumber(num) {
  isPrime(num)
    ? console.log(`${num} is a prime number.`)
    : console.log(`${num} is not a prime number.`);
}

PrimeNumber(17);
PrimeNumber(12);
PrimeNumber(13);
PrimeNumber(30);
